admin/views/partial-alerts-template.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+if ( ! function_exists( '_yoast_display_alerts' ) ) {
```

Comment:
To prevent duplicating code and keeping all the styling/structure inside the 'view' which handles the displaying only.
All HTML is now contained in a single place instead of having it laying around inside a class.